### DISCUSSION TITLE
Offline

### DISCUSSION THREAD

**Discussion by Nicolas-Loisy** [kkrisz1/staruml-reverse-db#138] [Q&A] [answered]
Does the extension work completely offline?

**Comment by kkrisz1** [comment, marked as answer]
yes if the targeted database is on your device (laptop, PC, etc.)

in other words databse must be reachable. no other dependencies are required (no other connections)

pls let me know if you experienced something strange :)

**Reply by Nicolas-Loisy** [reply]
Thx 👍The extension is very practical
